tests/php/Dev/CsvBulkLoaderTest.php

```diff
@@ -57,8 +57,8 @@ public function testLoad()
         $filepath = $this->csvPath . 'PlayersWithHeader.csv';
         $file = fopen($filepath ?? '', 'r');
         $compareCount = $this->getLineCount($file);
-        fgetcsv($file); // pop header row
-        $compareRow = fgetcsv($file);
+        fgetcsv($file, escape: "\\"); // pop header row
```

Comment:
Fixed Deprecated: fgetcsv(): the $escape parameter must be provided as its default value will change